### PR TITLE
Fix metronome double tick

### PR DESCRIPTION
**Pull request by UGEcko**
Fixed metronome double tick
Also changed some spots to use `metronomeActive` instead of `metronomeVolume`

### CODE REVIEW

**Comment by XAce1337manX** [comment]
For next time should drop or squash the commits that you don't need. (The second just reverts the first.)

Can squash merge this on the weekend after I merge v4.

**Comment by UGEcko** [comment]
Yeah I apologize, I didnt intend on committing a file that was changed on the first commit, I thought the last commit would be the latest.
Also thank :)

File: Assets/__Scripts/MapEditor/Audio/MetronomeHandler.cs
Comment (XAce1337manX) [inline Assets/__Scripts/MapEditor/Audio/MetronomeHandler.cs:95]: Can probably replace this while we're here
```suggestion
        if (!metronomeActive) return;
```

**Comment by UGEcko** [comment]
Oh yea good catch

**Review by XAce1337manX — APPROVED** [review]

**Pull request merged**
